MACHINE_LEARNING/machine_learning/models/main.py

```diff
@@ -9,17 +9,13 @@
 # accordance with the terms of the license agreement you entered into
 # with Jalasoft.
 #
-from prediccion import Prediccion
+from prediction import Prediction
 
-# noinspection SpellCheckingInspection
 if __name__ == '__main__':
-    # Definir el modelo de detección de objetos ("nasnet", "vgg", "resnet")
```

Comment:
Please translate the comments into english

Reply:
That line was removed. So is not necessary to translate that line.